examples/acquiring_data/fido_metadata_queries.py

```diff
@@ -0,0 +1,53 @@
+"""
+====================================
+Querying Metadata clients using Fido
+====================================
+
+This example shows how to search and retrieve metadata using `~sunpy.net.Fido` from the
+search facilities like `~sunpy.net.hek.HEKClient`, `~sunpy.net.helio.HECClient`,
+and `~sunpy.net.jsoc.JSOCClient`. It also shows how to display desired columns in the result.
+"""
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###################################################################
+# We will query Helio for the 'rhessi_flare_list' table and
+# shall limit the total number of records to 5.
+# For the same time range, we will query HEK for 'FL' as the
+# Event Type and 'PeakFlux' greater than 1000.
+# We will also search JSOC for 'hmi.m_45s' Series.
+timerange = a.Time('2010/8/1 03:40', '2010/8/1 3:40:10')
+results = Fido.search(timerange, a.helio.TableName('rhessi_hxr_flare') |
+                      a.hek.FL & (a.hek.FL.PeakFlux > 1000) |
+                      a.jsoc.Series('hmi.m_45s') & a.jsoc.Notify("jsoc@cadair.com"))
+print(results)
```

Comment:
I think there might need to be more description here on the structure of the results here.

Reply:
I have added some description below it.